src/wp-includes/html-api/class-wp-html-attribute-token.php

```diff
@@ -47,22 +51,36 @@ class WP_HTML_Attribute_Token {
 	 * The string offset where the attribute name starts.
 	 *
 	 * @since 6.2.0
+	 *
 	 * @var int
 	 */
 	public $start;
 
 	/**
-	 * The string offset after the attribute value or its name.
+	 * Byte length of the entire attribute name or name and value pair expression.
```

Comment:
Readability nitpick:

```suggestion
	 * Either:
	 * * A byte length of the entire attribute name, or
	 * * A name and value pair expression
```

Reply:
Thanks! I updated the wording in f2c715d621 to include different wording and now seeing this I've added f6d3a64d44